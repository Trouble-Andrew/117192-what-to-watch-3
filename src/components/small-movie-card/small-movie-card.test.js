import React from 'react';
import renderer from 'react-test-renderer';
import SmallMovieCard from "../small-movie-card/small-movie-card.jsx";


const film = {
  title: `Parasite`,
  date: `04.07.2019`,
  genre: [`Crime`, `Drama`, `Comedy`],
  poster: `https://m.media-amazon.com/images/M/MV5BYWZjMjk3ZTItODQ2ZC00NTY5LWE0ZDYtZTI3MjcwN2Q5NTVkXkEyXkFqcGdeQXVyODk4OTc3MTY@._V1_SY1000_CR0,0,674,1000_AL_.jpg`,
  posterBig: `https://m.media-amazon.com/images/M/MV5BNGVmNzE2ZmEtNTBmNC00MjVjLWE2ZjgtNmNhYjJlYTIzYjQ5XkEyXkFqcGdeQXVyNzgxMzc3OTc@._V1_SX1777_CR0,0,1777,999_AL_.jpg`,
  rating: 8.6,
  ratingCount: 227900,
  director: [`Bong Joon Ho`],
  stars: [`Kang-ho Song`, `Sun-kyun Lee`, `Yeo-jeong Jo`],
  preview: `A poor family, the Kims, con their way into becoming the servants of a rich family, the Parks. But their easy life gets complicated when their deception is threatened with exposure.`
};

it(`SmallMovieCard is rendered correctly`, () => {
  const tree = renderer.create(
      <SmallMovieCard movie={film} handleMouseEnterCard={() => {}} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
