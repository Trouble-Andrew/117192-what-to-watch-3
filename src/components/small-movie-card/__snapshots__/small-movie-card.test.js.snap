// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard is rendered correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/films/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Parasite"
        height={175}
        src="https://m.media-amazon.com/images/M/MV5BYWZjMjk3ZTItODQ2ZC00NTY5LWE0ZDYtZTI3MjcwN2Q5NTVkXkEyXkFqcGdeQXVyODk4OTc3MTY@._V1_SY1000_CR0,0,674,1000_AL_.jpg"
        width={280}
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <p
        className="small-movie-card__link"
      >
        Parasite
      </p>
    </h3>
  </a>
</article>
`;
